src/app/app.ts

```diff
@@ -1261,11 +1266,12 @@ function stepclick(): void {
     document.getElementById("sleep-img").style.color = '#999393';
     document.getElementById("water-img").style.color = '#999393';
     document.getElementById("food-img").style.color = '#999393';
-    document.getElementById("title").innerHTML = 'Steps Taken <span id="steptitle-annotation1">Distance Travelled</span>';
-    document.getElementById("subtitle").innerHTML = '<span id="stepsubtitle-annotation1" style="color: #05AD13;">3.2 miles</span>'
-    document.getElementById("pie-title").innerHTML = 'Today';
+	document.getElementById("pie-title").innerHTML = 'Today';
+	document.getElementById('chart-header-title').innerHTML = ' <p class="steps-ch-title" id="title"> Steps Taken </p>'
+	document.getElementById('total-value').innerHTML = '   <p class="steps-total-title" id="title"> Distance Travelled </p> <p class="steps-st-today subtitle" id="chart-steps-total-subtitle"> 3.2 miles </p>'
```

Comment:
remove the unwanted white space in inner HTML

Reply:
Unwanted white spaces in inner HTML has been removed